packages/zoe/zoe.chainmail

```diff
@@ -27,82 +28,66 @@ interface Zoe {
    */
   getInviteAssay() -> (Assay);
 
-  /**
-   * Get the long-lived escrowReceiptAssay. The mint associated
-   * with the escrowReceiptAssay creates the ERTP payments that
-   * represent proof of escrow and may be passed into a
-   * smart contract method.
-   */
-  getEscrowReceiptAssay() -> (Assay);
-
-  /**
-   * Returns the array of assays for the particular instance. This is 
-   * helpful in the case that the user has forgotten the order.
-   */
-  getAssaysForInstance() -> (List(Assay));
-
   /**
    * Create an installation by safely evaluating the code and
    * registering it with Zoe. Returns an installationHandle.
    */
-  install(code :String) -> Object
-
+  install(code :String, moduleFormat :String) -> (Object);
+  
   /** 
    * Zoe is long-lived. We can use Zoe to create smart contract
    * instances by specifying a particular contract installation to 
    * use, as well as the `terms` of the contract. The contract terms 
    * are the arguments to the contract, and must include
    * the expected assays for the underlying rights. (Other than the
    * `assays` property of `terms`, the `terms` properties are up to 
-   * the discretion of the smart contract.) We get back an instance, a
-   * handle for that instance, the handle for the installation, and the
-   * terms.
+   * the discretion of the smart contract.) We get back an invite (an
+   * ERTP payment) to participate in the contract.
    */
-  makeInstance(installationHandle :Object, terms :Object) -> (InstanceInformation);
```

Comment:
The old return type is still referenced in autoswap, covered-call, public-auction, public-swap and simple-exchange readme.md's. This looks like it affects the descriptions in the files that use .makeInstance in their examples. Is there updated literature?